src/legacy/core_plugins/telemetry/server/telemetry_config/get_telemetry_notify_user_about_optin_default.ts

```diff
@@ -41,5 +41,9 @@ export function getNotifyUserAboutOptInDefault({
     return false;
   }
 
-  return telemetryOptedIn === true && configTelemetryOptIn === true;
+  if (telemetryOptedIn !== null) {
+    return false; // opted in by default
```

Comment:
nit: might want to say: `hasn't made a telemetry opt-in/out` in the comment

Reply:
942089a476